scripts/scil_dwi_reorder_philips.py

```diff
@@ -55,9 +63,21 @@ def main():
                         args.out_basename + '.bval',
                         args.out_basename + '.bvec']
 
-    assert_inputs_exist(parser, required_args)
+    assert_inputs_exist(parser, required_args, args.json)
     assert_outputs_exist(parser, args, output_filenames)
 
+    if args.json and args.force:
+        with open(args.json) as curr_json:
+            dwi_json = json.load(curr_json)
+        if 'SoftwareVersions' in dwi_json.keys():
+
+            curr_version = dwi_json['SoftwareVersions']
+            curr_version = curr_version.replace('\\', ' ').replace('_', ' ')
```

Comment:
Replace this line by `curr_version = curr_version.split('_')[0]`.

Reply:
Ok then put 
`curr_version = curr_version.replace('\\', '_ ')`
`curr_version = curr_version.split('_')[0]`